cmd/kubeadm/app/cmd/phases/certs_test.go

```diff
@@ -20,11 +20,13 @@ import (
 	"os"
 	"testing"
 
+	"github.com/spf13/cobra"
 	kubeadmapi "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm"
 	"k8s.io/kubernetes/cmd/kubeadm/app/cmd/phases/workflow"
 	"k8s.io/kubernetes/cmd/kubeadm/app/phases/certs"
 	"k8s.io/kubernetes/cmd/kubeadm/app/util/pkiutil"
 	testutil "k8s.io/kubernetes/cmd/kubeadm/test"
+	certstestutil "k8s.io/kubernetes/cmd/kubeadm/test/certs"
```

Comment:
so i just noticed that we are doing this pattern. :[
phases and such are eventually going to be part of the vendorable backend of kubeadm.

`cmd/kubeadm/test` hold our integration tests it should not to be imported into the "backend", it should be the other way around.

i think the location for such test utils is ideally inside the backend itself, but given we are following a previous pattern this is fine i guess.

Reply:
That file could be anywhere, it just happened to be where it was picked. A future PR could very easily move it I think, but that's probably out of scope

Reply:
Filed as kubernetes/kubeadm#1257